### ISSUE TITLE
What is the rationale behind PT001?

### ISSUE THREAD

**Issue by d1618033** [m-burst/flake8-pytest-style#4]
In the official pytest documentation there are examples using `pytest.fixture` and not `pytest.fixture()`

https://docs.pytest.org/en/latest/fixture.html
```python
# content of ./test_smtpsimple.py
import pytest


@pytest.fixture
def smtp_connection():
    import smtplib

    return smtplib.SMTP("smtp.gmail.com", 587, timeout=5)


def test_ehlo(smtp_connection):
    response, msg = smtp_connection.ehlo()
    assert response == 250
    assert 0  # for demo purposes
```

**Comment by m-burst** [comment]
Hi @d1618033,

My intention for PT001 was primarily to enforce consistency between all tests in a single codebase (i.e. to disallow having `@pytest.fixture()` in one test and `@pytest.fixture` in another).
I understand that the rule is opinionated, but at the moment of writing the rule I wanted to keep it simple, so I chose to enforce the syntax that my team at work agreed upon.
I might make the rule configurable in the future if I have time (or I would gladly accept a PR).

**Comment by m-burst** [comment]
Hi @d1618033,
The configuration option for PT001 is now available in v0.5.0.

**Comment by d1618033** [comment]
Thank you!
